src/main.rs

```diff
@@ -54,6 +54,18 @@ struct Cli {
     /// GGA update period, in seconds. 0 means to never send a GGA
     #[clap(long, default_value = "10")]
     gga_period: u64,
+
+    /// AreaID to be used in generation of CRA message. If this flag is set, ntripping outputs messages of type CRA rather than the default CRA.
```

Comment:
```suggestion
    /// Area ID to be used in generation of CRA message. If this flag is set, ntripping outputs messages of type CRA rather than the default CRA.
```

Reply:
Done